yaksh/views.py

```diff
@@ -505,27 +556,42 @@ def start(request, questionpaper_id=None, attempt_num=None, course_id=None,
 @login_required
 @email_verified
 def show_question(request, question, paper, error_message=None, notification=None,
-                  course_id=None, module_id=None):
+                  course_id=None, module_id=None, previous_question=None):
     """Show a question if possible."""
     user = request.user
+    quiz = paper.question_paper.quiz
+    quiz_type = 'Exam'
+    can_skip = False
+    if previous_question:
```

Comment:
Same with timed questions. If the ```duration``` for exercise is set to a time smaller than the question duration, some how clicking on the submit button does nothing.

Reply:
This works. Actually submit button won't go to the next question.
User has to click on Next to visit next question.